__tests__/app.test.js

```diff
@@ -42,6 +42,39 @@ describe('/api/articles/:article_id', () => {
     })
 })
 
+describe('/api/articles', () => {
+    test('GET: 200 - responds with the article and properties', () => {
+        return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then((response) => {
+                expect(typeof response.body).toBe('object')
+                expect(response.body.length).toBe(13)                
+                response.body.forEach(article => {
+                    expect(article).toHaveProperty('author')
+                    expect(article).toHaveProperty('title')
+                    expect(article).toHaveProperty('article_id')
+                    expect(article).toHaveProperty('topic')
+                    expect(article).toHaveProperty('created_at')
+                    expect(article).toHaveProperty('votes')
+                    expect(article).toHaveProperty('article_img_url')
+                    expect(article).toHaveProperty('comment_count')
+                    expect(article).not.toHaveProperty('body')
```

Comment:
also test for the type of the data expected i.e. .toHaveProperty('title', expect.any(String))

I like you .not.toHaveProperty('body') this is nicely done

Reply:
Great